platform/darwin/test/MGLFunctionTests.m

```diff
@@ -0,0 +1,13 @@
+#import <XCTest/XCTest.h>
+#import <Mapbox/Mapbox.h>
+
+@interface MGLFunctionTests : XCTestCase
```

Comment:
MGLStyleValueTests is taken by a Swift class with the same name but I can't use that class to catch a specific exception name if the method itself doesn't throw the exception. Renamed it to MGLStyleValueObjCTests for now.